src/applications/personalization/dashboard/sass/dashboard-alert.scss

```diff
@@ -31,24 +54,35 @@
 
   section.status {
     margin: 1.5em 0;
-    padding: 1em 0 1em 2em;
+    padding: 1em 0;
```

Comment:
We're trying to start using consistent rhythms in horizontal/vertical spacing. Any chance you'd be able to use the spacing tokens instead? https://design.va.gov/design/spacing-units

These are really useful on margins and padding, but can also be set to width and height in some instances as well.

Reply:
I just pushed up a change that addresses this.